### ISSUE TITLE
Debate: Defaulting mobile to Perspective 0

### ISSUE THREAD

**Issue by humanat** [NoHatCoder/Playtak#21]
This throws people off, and most new experiences with it don't realize that it's intentional and controllable from the interface settings.

We should probably leave the default at 90 regardless of screen size.

**Comment by NoHatCoder** [comment]
The thing is, rotating the board works badly on touch devices, even if the technical stuff was consistent it is still a bit too many features all operated by touch for a lot of people, thus fixing the view makes the experience better. But in fixed view with perspective some stacks may be hidden, the orthographic projection makes all stacks at least partially visible. So for a different perspective you would also have to enable rotations, and I don't think that is a good idea. Great if it works for you, but I don't currently see a path to a better default.

**Comment by humanat** [comment]
Yeah, I’m not sure there’s a good option here other than maybe a popup to point people to the option.  Otherwise people think it’s just a bug in the rendering.
